### ISSUE TITLE
Ability to set store-id

### ISSUE THREAD

**Issue by misilot** [DSpace/dspace-replicate#73]
Maybe I am missing it, but I do not believe there is a way to set the `store-id` flag for the duracloud sync portion.

I believe this is needed to be able to upload directly to Chronopolis which is hosted on the same platform as Duracloud.

**Comment by nwoodward** [comment]
You're correct that there's currently no way to set the `store-id`. The RTS makes a call to DuraCloud to get the primary storage provider (https://github.com/DSpace/dspace-replicate/blob/main/src/main/java/org/dspace/ctask/replicate/store/DuraCloudObjectStore.java#L81) and uses that.

I think it would be fairly straightforward to make that configurable by adding a parameter and getting the desired ContentStore using the getContentStore method in the ContentStoreManager: https://github.com/duracloud/duracloud/blob/develop/storeclient/src/main/java/org/duracloud/client/ContentStoreManager.java#L61-L62.

**Comment by nwoodward** [comment]
We are now using this `store-id` setting in production on the 8.1 release. We can use it to transmit DSpace content to spaces in storage providers that aren't the primary.
